homework_1/zad3/pokemon/findLocations.go

```diff
@@ -0,0 +1,80 @@
+package pokemon
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/pkg/errors"
+	"github.com/sethgrid/pester"
+	"io/ioutil"
+	"log"
+)
+
+const pokemonData = "https://pokeapi.co/api/v2/pokemon/"
+
+type LocationArea struct {
+	Name string
+}
+
+type LocationAreaEncounter struct {
+	Location LocationArea `json:"location_area""`
+}
+
+type Pokemon struct {
+	Name string `json:"name"`
+}
+
+type Output struct {
+	Name string
+	Locations []string
+}
+
+func GetData(url string) []byte {
+	httpClient := pester.New()
+
+	httpResponse, err := httpClient.Get(url)
+	if err != nil {
+		log.Fatal(
+			errors.WithMessage(err, "HTTP get towards pokemon API"),
+		)
```

Comment:
Do not use `log.Fatal` (except in the main program).
Doing that will cause the program to crash, which is definitely not desired from the perspective of the main program. The best approach is to return errors everywhere and in the end handle those errors from the main program.